src/Middleware/ConcurrencyLimiter/sample/ConcurrencyLimiterSample.csproj

```diff
@@ -1,21 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
-
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(BenchmarksTargetFramework)' == ''">
+  <ItemGroup>
```

Comment:
@sebastienros to confirm: Removing this `Condition` won't cause problems for the benchmarking infrastructure e.g. conflicts w/ what's in the shared framework❔

Reply:
Right this one should be kept, only the <FrameworkREference> should be removed. And keep that too:

```
<ItemGroup Condition="'$(BenchmarksTargetFramework)' != ''">
    <PackageReference Include="Microsoft.AspNetCore.ConcurrencyLimiter" Version="$(MicrosoftAspNetCoreAppVersion)" />
</ItemGroup>
    
```

Reply:
The idea is that `<Reference>` is not recognized by the benchmarks, so in that case we just point to the matching package.